### DISCUSSION TITLE
Sponsoring Lucide Icons

### DISCUSSION THREAD

**Discussion by ben519** [lucide-icons/lucide#2089] [General]
Hey, I use lucide icons for my Markdown-based writing platform, [Scipress.io](https://www.scipress.io/). This repo has been super valuable to my project. As such, I'd like to make a monetary donation to support the community in developing and maintaining lucide icons.

My ask is to feature my company logo with a link to my site on the home README, similar to the "Powered by Vercel" and "Powered by Digital Ocean" links.

Please let me know if this is possible and if there's a dollar threshold to make it onto the README. Thanks!

**Comment by ericfennis** [comment]
Hi @ben519!

Cool project, looks really nice!
Thanks for using lucide for this.

We have our sponsoring page on [Open Colletive](https://opencollective.com/lucide-icons), I've added some tiers to get mentioned in our README file. I hope this what you wanted.

Thanks for considering sponsoring the Lucide project ❤️

**Reply by ben519** [reply]
Hey, I just made a contribution. Let me know what I need to do to get on the README. Thanks!

**Reply by ericfennis** [reply]
@ben519 Thank you so much! We appreciate it!
Can you send me the image and link you want to have for the README?

**Reply by ericfennis** [reply]
I've added an image and a URL to your project in our readme files.

**Reply by ben519** [reply]
Sorry for the late reply. Awesome, thanks for setting that up! Keep up the good work.

**Reply by ben519** [reply]
Hey @ericfennis any chance you could look again at [this issue](https://github.com/lucide-icons/lucide/issues/1660) that was auto closed? It's something I could really use and I noticed that other people wanted it too. Thanks!

**Reply by ericfennis** [reply]
I've reopened it. Yess we will take a look!
